projects/packages/my-jetpack/_inc/components/modules-list/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { Flex } from '@wordpress/components';
+import { DataViews, Field } from '@wordpress/dataviews';
+import { __ } from '@wordpress/i18n';
+import { useMemo } from 'react';
+import { ModuleStatus } from '../module-status';
+import { ModuleToggle } from '../module-toggle';
+import { MyJetpackModule } from '../types';
+import styles from './styles.module.scss';
+
+import './style.scss';
+
+export type ModulesListProps = {
+	modules: Array< MyJetpackModule >;
+};
+
+const noop = () => {};
```

Comment:
It would be beneficial to define noop at the project level, rather than define it locally for every time we want to use it - what do you think?

Reply:
Doesn't hurt but i am fine with that if we have a ready to use function somewhere.